ballot/models.py

```diff
@@ -52,6 +52,10 @@ def __str__(self):
 class BallotItemResponse(models.Model):
     """
     YES/NO to a referendum, or a candidate.
+
+    This is a *conceptually* abstract class, but exists
+    as a Django model so that finance can point to propositions
+    and candidates equally.
     """
```

Comment:
You're not a pill at all, this is wonderful feedback. I really appreciate it, and appreciate your patience to walk through it with me--I also feel I'm being a pill :)

This is subclassed for candidates here:
https://github.com/caciviclab/disclosure-backend/blob/master/office_election/models.py#L101

So, just to be clear, it's definitely not a response from each person. It's a potential response--a bubble you could fill on the ballot, an answer to a specific ballot item question.

`BallotItemDescription` sounds like a description of the entire ballot item. This is not that. There is one `BallotItemResponse` per possible responses to the ballot item, and each has information related to that response (title: yes/no/caniddate name, information in favor / opposing that particular response). So for each `BallotItem`, you need multiple `BallotItemResponse` to fully describe the entire item on the ballot.

As far as fields on the `BallotItemResponse` object, I agree that it's very much structured towards a referendum. I think it can be abstracted better. Lemme think about use-cases, but I think moving most of the properties onto `Candidate` and `ReferendumResponse` is a good idea.